boards/common/nrf52/Makefile.dep

```diff
@@ -0,0 +1,9 @@
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += saul_gpio
+endif
+
+ifeq (,$(filter nrfmin,$(USEMODULE)))
+  ifneq (,$(filter gnrc_netdev_default,$(USEMODULE)))
+    USEPKG += nordic_softdevice_ble
```

Comment:
Apparently there is an order issue with the `USEPKG` here and the` ifneq (,$(filter nordic_softdevice_ble,$(USEPKG)))` in `Makefile.include`
see discussion in #8068

Reply:
yep, so you think we should wait for #8068 before going on with this PR?

Reply:
This is not mandatory, we can merge this one "as it", I'll rebase #8068 and try to fix this issue since it is related to the softdevice. But if you already have a potential fix you can add it there.